src/app/Plans/components/Wizard/FilterVMs.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Form, FormGroup, TreeView, Title } from '@patternfly/react-core';
+
+// TODO replace with redux or equiv.
+import { VMsOptions } from './mocks/VMs.mock';
+
+interface IFilterVMsProps {
+  Inventory: [];
+}
+
+const FilterVMs: React.FunctionComponent<IFilterVMsProps> = ({ Inventory }: IFilterVMsProps) => {
+  const [activeItems, setItems] = React.useState(VMsOptions);
+  const [checkedItems, setCheckedItems] = React.useState([]);
+
+  const onSelect = (event, treeViewItem, parentItem) => {
+    return;
+  };
+
+  const onCheck = (event, treeViewItem) => {
```

Comment:
Is this a potential use case for a reusable hook?

Reply:
Good point - we need to make sure we are operating in a way that is react friendly.

Reply:
@gildub let's make this `onCheck` function a no-op for now, I don't want to leave the imperative mutation here just in case we forget about it.